examples/vite/src/App.tsx

```diff
@@ -70,10 +72,15 @@ const App = () => {
     userData: { id: userId },
   });
 
+  const isMessageAIGenerated = useCallback(
+    (message: StreamMessage<StreamChatGenerics>) => !!message?.ai_generated,
+    [],
+  );
```

Comment:
I think we can move this out of the `App` component completely.

Reply:
Moved out: https://github.com/GetStream/stream-chat-react/pull/2588/commits/fc2a860ded11417fd6b4f4017a76191a69223f5f